react/src/lib/components/SubsurfaceViewer/components/Map.tsx

```diff
@@ -328,15 +333,25 @@ const Map: React.FC<MapProps> = ({
     const [viewsProps, setViewsProps] = useState<ViewportType[]>([]);
     const [alteredLayers, setAlteredLayers] = useState<LayersList>([]);
 
+    // Viewport margins. Will differ from 0 if axes2Dlayer is used.
+    const [marginLeft, setMarginLeft] = useState<number>(0);
+    const [marginBottom, setMarginBottom] = useState<number>(0);
+
     const initialViewState = getViewState(
```

Comment:
Unrelated, but I think this could benefit from being memoized, ie. `const initialViewState = useMemo(() => {...}, [...])`